app/components/layouts/BackNavigation.tsx

```diff
@@ -6,22 +6,23 @@ import { useHistory } from "react-router-dom";
 // Renders a smart back link that handles pages visited directly or from referring page
 export const BackNavigation = ({
   defaultReturnTo,
+  content,
 }: {
-  defaultReturnTo: string;
+  defaultReturnTo?: string;
+  content: string;
 }) => {
   const history = useHistory();
 
   // The "POP" action is reserved as a default action for newly created history objects. `react-router-dom` depends on
   // another history library that defines this behavior. Read more:
   // https://github.com/remix-run/history/blob/main/docs/api-reference.md#reference
-  const backDestination =
-    history.action === "POP"
-      ? () => history.push(defaultReturnTo)
-      : () => history.goBack();
+  const backDestination = defaultReturnTo
+    ? () => history.push(defaultReturnTo)
+    : () => history.goBack();
```

Comment:
I think this simplification works. However, in order to cover all use cases as defined by our [decision table](https://www.notion.so/exygy/Back-Navigation-Bar-2564a5fd669e46f79eeba5ec61ff3c47?pvs=4#cc7103ec8d444f19ba17872822629ad5), let's make `defaultReturnTo` required.